elements/lisk-transactions/test/14_unlock_transaction.spec.ts

```diff
@@ -13,120 +13,800 @@
  *
  */
 
+import * as validUnlockTransactionScenario from '../fixtures/unlock_transaction/unlock_transaction.json';
+
+import { UnlockTransaction } from '../src/14_unlock_transaction';
+import { Status, Account } from '../src';
+import { StateStoreMock, defaultAccount } from './utils/state_store_mock';
+import { AccountUnlocking } from '../src/transaction_types';
+import { sortUnlocking } from '../src/utils';
+
 describe('Unlock transaction', () => {
+	const minBalance = BigInt('5000000');
+	let tx: UnlockTransaction;
+
+	beforeEach(async () => {
+		tx = new UnlockTransaction({
+			...validUnlockTransactionScenario.testCases.output,
+			networkIdentifier:
+				validUnlockTransactionScenario.testCases.input.networkIdentifier,
+		});
+	});
+
 	describe('validateAsset', () => {
 		describe('when asset.votes contains valid contents', () => {
-			it.todo('should not return errors');
+			it('should not return errors', async () => {
+				const { errors, status } = tx.validate();
+				expect(status).toBe(Status.OK);
+				expect(errors).toHaveLength(0);
+			});
 		});
 
 		describe('when asset.unlockingObjects does not include any unlockingObject', () => {
-			it.todo('should return errors');
+			it('should return errors', async () => {
+				(tx.asset as any).unlockingObjects = [];
+				const { errors, status } = tx.validate();
+				expect(status).toBe(Status.FAIL);
+				expect(errors).toHaveLength(1);
+				expect(errors[0].message).toInclude(
+					'should NOT have fewer than 1 items',
```

Comment:
the assertion is bit confusing. when there are no `unlockingObjects` the assertion should be `should have fewer than 1 items`?

Reply:
it's expecting the error, so this is correct?